### DISCUSSION TITLE
Is it possible to simulate drag events on parent from iframe so that dockview will pick them up?

### DISCUSSION THREAD

**Discussion by kosaj** [mathuo/dockview#332] [Q&A]
Hey, I have groups (with dockview tabs set to hidden) that contain iframes with draggable header (inside iframe). I'm trying to simulate (using dispatchEvent) the dragging behaviour from within the iframe sending (dragstart/dragend/drag) data over postMessage to the parent window but I'm unable to trigger overlay/onDidDrop on DockviewReact. Mby someone tried to achieve something similar and might guide me in the right direction?

I looked what events (dragenter/dragover/dragleave/dragend/drop) are used by [dockview dnd](https://github.com/mathuo/dockview/blob/63ca9053a5e121f8afc03f9f0819ae548a06c1d2/packages/dockview-core/src/dnd/dnd.ts)  and tried to simulate them with 'dispatchEvent'  and a mix of

```
document.elementsFromPoint(clientX, clientY)
            .filter((element) => element.id === 'plugin-container');

          Object.defineProperty(dragEvent, 'clientX', {
            get: () => clientX,
          });
          Object.defineProperty(dragEvent, 'clientY', {
            get: () => clientY,
          });

          Object.defineProperty(dragEvent, 'target', {
            value: target,
          });
```

**Comment by mathuo** [comment]
I can take a look at this. To confirm, you are trying to take a drag event that originates in an iframe and intercept the drop event within dockview?
